packages/web-app-epub-reader/src/App.vue

```diff
@@ -222,7 +221,7 @@ export default defineComponent({
         })
 
         unref(rendition).on('relocated', () => {
-          const currentLocation = unref(rendition).currentLocation() as DisplayedLocation & Location
+          const currentLocation = unref(rendition).currentLocation() as any & Location
```

Comment:
Oh no :-( aren't they exporting the type anymore or what's the problem?

Reply:
It was never exported in the types.ts but in the the types/redention which seems not to be accessible when built

Reply:
😢